home_audit.node.js

```diff
@@ -1169,7 +1169,7 @@ let validationRules = {
      */
     _installation_year: function(value, minYear) {
         if(parseInt(value) >= minYear) {
-            return new Validation(TypeRules._int(value, parseInt(_homeValues.year_built), (new Date()).getFullYear()), ERROR);
+            return new Validation(TypeRules._int(value, parseInt(_homeValues.year_built) || 1970, (new Date()).getFullYear()), ERROR);
```

Comment:
I think this is fixing the problem in the wrong place. Instead of passing some arbitrary value here, why don't we change the `Validation` constructor so that it handles non-integer min/max values like `NaN` in a way that we like instead of having to add handling anywhere we pass those arguments?

Reply:
That was my initial thought, but it still didn't seem quite right to do it that way.  

What would we parse it to?  We can't just set it to zero, because in this case, <1970 won't be accepted by the API, and what if we have negative boundaries someday?  

We could add error handling inside, but then we're returning an error here just for the case where a user forgets an assessment_date, which also seems wrong.